.gitignore

```diff
@@ -20,6 +20,7 @@ go.list
 
 # Jetbrains
 .idea/
+*.iml
```

Comment:
It's nothing special and doesn't block the whole PR, but I would suggest to make .gitignore changes is separate PR.